tools/dependencies/known-dependencies.txt

```diff
@@ -313,8 +314,11 @@ spring-jcl-5.3.22.jar
 spring-jdbc-5.3.22.jar
 spring-ldap-core-2.4.1.jar
 spring-retry-1.3.3.jar
+spring-security-config-5.7.3.jar
```

Comment:
Please also add these dependencies' license to this file 

https://github.com/apache/dolphinscheduler/blob/dev/dolphinscheduler-dist/release-docs/LICENSE